### ISSUE TITLE
v0.6.1 is broken; can't import fsspec.http.BlockSizeError

### ISSUE THREAD

**Issue by rabernat** [pangeo-forge/pangeo-forge-recipes#273]
In this [workflow](https://github.com/pangeo-data/pangeo-cmip6-cloud/runs/5062065294?check_suite_focus=true) I create a fresh python 3.9 env and `pip install pangeo_forge_recipes`. This  error is raised when trying to import pangeo_forge_recipes

```
Traceback (most recent call last):
  File "/home/runner/work/pangeo-cmip6-cloud/pangeo-cmip6-cloud/zarr_from_esgf.py", line 6, in <module>
    from pangeo_forge_recipes.storage import FSSpecTarget, CacheFSSpecTarget, MetadataTarget
  File "/opt/hostedtoolcache/Python/3.[9](https://github.com/pangeo-data/pangeo-cmip6-cloud/runs/5062065294?check_suite_focus=true#step:5:9).[10](https://github.com/pangeo-data/pangeo-cmip6-cloud/runs/5062065294?check_suite_focus=true#step:5:10)/x64/lib/python3.9/site-packages/pangeo_forge_recipes/storage.py", line 16, in <module>
    from fsspec.implementations.http import BlockSizeError
ImportError: cannot import name 'BlockSizeError' from 'fsspec.implementations.http' (/opt/hostedtoolcache/Python/3.9.10/x64/lib/python3.9/site-packages/fsspec/implementations/http.py)
Error: Process completed with exit code 1.
```

**Comment by rabernat** [comment]
Changing it to `pip install git+https://github.com/pangeo-forge/pangeo-forge-recipes.git` turned up another error.

```
Traceback (most recent call last):
  File "zarr_from_esgf.py", line 8, in <module>
    from pangeo_forge_recipes.recipes import XarrayZarrRecipe
  File "/opt/hostedtoolcache/Python/3.8.[12](https://github.com/pangeo-data/pangeo-cmip6-cloud/runs/5062105560?check_suite_focus=true#step:5:12)/x64/lib/python3.8/site-packages/pangeo_forge_recipes/recipes/__init__.py", line 1, in <module>
    from .reference_hdf_zarr import HDFReferenceRecipe
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pangeo_forge_recipes/recipes/reference_hdf_zarr.py", line 12, in <module>
    from ..executors.base import Pipeline, Stage
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pangeo_forge_recipes/executors/__init__.py", line 1, in <module>
    from .python import FunctionPipelineExecutor, GeneratorPipelineExecutor
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pangeo_forge_recipes/executors/python.py", line 5, in <module>
    from .base import Pipeline, PipelineExecutor
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pangeo_forge_recipes/executors/base.py", line 7, in <module>
    from mypy_extensions import NamedArg
ModuleNotFoundError: No module named 'mypy_extensions'
```

**Comment by cisaacstern** [comment]
Yes, getting the BlockSizeError fix from #247 released is a priority. Seems like you found another bug to fix as well.

In addition, maybe we should get #249 fixed before releasing? I made a start at that in #271, but don't yet know why the tests are failing there (seems like a workflow caching issue, not a pytest failure).

**Comment by rabernat** [comment]
Fixed by #247
